test/counter_wintf_test.dart

```diff
@@ -86,4 +97,24 @@ void main() {
       expect(simResult, equals(true));
     });
   });
+  test('resetFlipflop from root', () async {
+    final mod = Counter(CounterInterface(8), resetRoot: true);
```

Comment:
instead of copy-pasting the whole test, you could make a function that generates a `test` with vs without `resetRoot` set.